python_modules/dagster/dagster/_daemon/asset_daemon.py

```diff
@@ -509,8 +524,20 @@ def _run_iteration_impl(
                         )
 
                     set_has_migrated_to_sensors(instance)
+                if not get_has_migrated_sensor_names(instance):
```

Comment:
just confirming my understanding - for new users who have never used AMP before or had a sensor before, the only thing this does is set a key in the kvs table right? Nothing gets moved or migrated in that case?

Reply:
that's correct yep -- it will not find any historical instigator data in the database and so no migration is actually happening